crates/forge-runner/src/running.rs

```diff
@@ -358,3 +378,31 @@ fn get_latest_block_number(url: &Url) -> Result<BlockId> {
         _ => Err(anyhow!("Could not get the latest block number".to_string())),
     }
 }
+
+fn extend_execution_resources(test_execution_syscall_handler: &mut TestExecutionSyscallHandler) {
```

Comment:
I don't really know what this function does from the name, and neither from the code